primary/tests/epoch_change.rs

```diff
@@ -465,6 +469,8 @@ async fn test_restart_with_new_committee_change() {
 
         // Wait for the committee to shutdown.
         join_all(handles).await;
+        // Provide a small amount of time for any background tasks to shutdown
+        tokio::time::sleep(Duration::from_secs(1)).await;
```

Comment:
The network shuts down when all the handles to it are dropped. After the join above we we need some delay for the background network task to unbind from the network port otherwise the test will fail when we try to startup a new network and bind on the same port.